server/templates/custom_dc/worldbank/homepage.html

```diff
@@ -24,17 +24,18 @@
 {% set is_hide_sub_footer = true %}
 
 {% block head %}
-<link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['VERSION'])}}>
+<link rel="stylesheet" href={{url_for('static', filename='css/homepage.min.css', t=config['VERSION'])}}>
+ <script src={{url_for('static', filename='homepage_custom_dc.js', t=config['VERSION'])}} async></script>
```

Comment:
nit: make these two lines `<` aligned. I think there's an extra space in front of `<script`.

Reply:
Done